src/coreclr/jit/instr.h

```diff
@@ -418,8 +418,10 @@ enum emitAttr : unsigned
                 EA_4BYTE         = 0x004,
                 EA_8BYTE         = 0x008,
                 EA_16BYTE        = 0x010,
-
-#if defined(TARGET_XARCH)
+#if defined(TARGET_ARM64)
+                EA_SCALABLE      = 0x020,
```

Comment:
Intention here is to use scalable alongside another size? For example, for an SVE vector of ints:
```csharp
    theEmitter->emitIns_R_R_R(INS_sve_and, EA_1BYTE|EA_SCALABLE, REG_V0, REG_V1, REG_V2, INS_OPTS_NONE);  // AND     <Zdn>.<T>, <Pg>/M, <Zdn>.<T>, <Zm>.<T>
```

Reply:
I think we need to add the following lane size enum entries in `insOpts` to represent number of lanes in scalable registers.
```
INS_OPTS_SCALABLE_B,
INS_OPTS_SCALABLE_H,
INS_OPTS_SCALABLE_S,
INS_OPTS_SCALABLE_D,
```

Then, for your example, it would be:

```c++
   theEmitter->emitIns_R_R_R(INS_sve_and, EA_SCALABLE, REG_V0, REG_V1, REG_V2, INS_OPTS_SCALABLE_B);  // AND     <Zdn>.<T>, <Pg>/M, <Zdn>.<T>, <Zm>.<T>
```